src/main/java/edu/cornell/kfs/coa/document/validation/impl/SubAccountGlobalPreRules.java

```diff
@@ -1,13 +1,18 @@
 package edu.cornell.kfs.coa.document.validation.impl;
 
-import edu.cornell.kfs.sys.CUKFSConstants;
+import org.apache.commons.lang3.StringUtils;
+import org.kuali.kfs.coa.businessobject.Account;
 import org.kuali.kfs.coa.document.validation.impl.MaintenancePreRulesBase;
+import org.kuali.kfs.kns.document.MaintenanceDocument;
+import org.kuali.kfs.krad.util.ObjectUtils;
 import org.kuali.kfs.sys.KFSConstants;
+import org.kuali.kfs.sys.KFSPropertyConstants;
```

Comment:
Is this import used?

Reply:
Good catch, it's not used so I'll remove it.